examples/platform/silabs/ldscripts/efr32mg26.ld

```diff
@@ -196,16 +206,24 @@ SECTIONS
     KEEP(*(SORT(.fini_array.*)))
     KEEP(*(.fini_array))
     PROVIDE_HIDDEN (__fini_array_end = .);
+    . = ALIGN(4);
+    *(SEGGER_RTT)
```

Comment:
Any idea why this SEGGER RTT is needed for the mg26 but not mg24?

Reply:
I don't know, that is what was generated by slc. I'll look.

Reply:
there was a missing segger rtt component for mg24 slc generation. I added it now